### PR TITLE
Additional admin page styles

### PR DESCRIPTION
**Pull request by tenshiemi**

### CODE REVIEW

File: client/src/admin/AlertForm.js [hunk 1]
```diff
@@ -3,28 +3,37 @@ import { Field, reduxForm } from 'redux-form';
...
-      </div>
-      <input type="submit" disabled={submitting || !(valid && featurePicked)} value="Send Alert!" />
-    </fieldset>
-  </form>
+  <div className="container--span">
+    <form onSubmit={handleSubmit}>
+      <legend className="legend--has-subtitle">
+        <h3>Enter a message describing the event:</h3>
+        <span className="text--subtitle">
+          Make sure to provide a link to allow the reader to get additional information.
```
Comment (macrael) [inline client/src/admin/AlertForm.js:17]: Can you integrate this message with the message I added in the last PR that provides a link to more information if we have it?

Comment (tenshiemi) [reply client/src/admin/AlertForm.js:17]: Sure!

File: client/src/admin/AdminHeader.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+const AdminHeader = () => (
+  <div>
+    <h1 className="text--center text__margin--70">Submit New Notification</h1>
+    <div className="container--blue">
+      On this page you can set up an alert for the following services
+      provided by Interior Geospatial Emergency Management System (IGEMS).
+      Subscribers will receive an alert via text or email.
```
Comment (macrael) [inline client/src/admin/AdminHeader.js]: "Subscribers in the selected area" I think. We need to make it explicit that they are supposed to click on a zone.

File: client/src/admin/EmergencyPicker.js [hunk 3]
```diff
@@ -35,6 +35,7 @@ class EmergencyPicker extends React.Component {
 
     return (
       <div>
+        <h3>Select the region affected by the incident:</h3>
```
Comment (macrael) [inline client/src/admin/EmergencyPicker.js]: "Select the region you wish to alert"

File: client/src/admin/AlertForm.js [hunk 4]
```diff
@@ -13,6 +19,15 @@ const AlertForm = ({ featurePicked, handleSubmit, submitting, valid }) => (
           while writing your message and use a link shortener when possible.
         </span>
       </legend>
+      { (feature && feature.properties.link) ?
+        <div className="container--span">
+          <p className="text--subtitle">
+          Use the following link provides additional information about the region you selected:
```
Comment (macrael) [inline client/src/admin/AlertForm.js]: s/Use t/T/

**Pull request merged**